package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I don't think this file is doing anything here (`package.json` is usually used for managing dependencies with npm in Javascript projects)